src/PrestaShopBundle/EventListener/AccessDeniedListener.php

```diff
@@ -64,6 +64,11 @@ public function onKernelException(ExceptionEvent $event)
         }
 
         $controllerName = $event->getRequest()->attributes->get('_controller');
+
+        if (!str_contains($controllerName, '::')) {
+            return;
+        }
```

Comment:
I added security for routes not including FQCN::method. This is the case here for API

Reply:
I removed it. Playing with an `event_listeners_backward_compatibility_layer: false` tag, it gave an exception because the controllerName had a strange format, like a route. I finally removed the tag, so no need